src/libs/GetPhysicalCardUtils.ts

```diff
@@ -91,23 +86,18 @@ function setCurrentRoute(currentRoute: string, domain: string, privatePersonalDe
  * @returns
  */
 function getUpdatedDraftValues(draftValues: DraftValues, privatePersonalDetails: PrivatePersonalDetails, loginList: LoginList) {
-    const {
-        address: {city, country, state, street = '', zip},
-        legalFirstName,
-        legalLastName,
-        phoneNumber,
-    } = privatePersonalDetails;
+    const {address: {city, country, state, street, zip} = {}, legalFirstName, legalLastName, phoneNumber} = privatePersonalDetails;
 
     return {
-        legalFirstName: draftValues.legalFirstName || legalFirstName,
-        legalLastName: draftValues.legalLastName || legalLastName,
-        addressLine1: draftValues.addressLine1 || street.split('\n')[0],
-        addressLine2: draftValues.addressLine2 || street.split('\n')[1] || '',
-        city: draftValues.city || city,
-        country: draftValues.country || country,
-        phoneNumber: draftValues.phoneNumber || (phoneNumber ?? UserUtils.getSecondaryPhoneLogin(loginList) ?? ''),
-        state: draftValues.state || state,
-        zipPostCode: draftValues.zipPostCode || zip,
+        legalFirstName: draftValues.legalFirstName ?? legalFirstName,
+        legalLastName: draftValues.legalLastName ?? legalLastName,
+        addressLine1: draftValues.addressLine1 ?? street?.split('\n')[0] ?? '',
+        addressLine2: draftValues.addressLine2 ?? street?.split('\n')[1] ?? '',
+        city: draftValues.city ?? city,
+        country: draftValues.country ?? country,
+        phoneNumber: draftValues.phoneNumber ?? phoneNumber ?? UserUtils.getSecondaryPhoneLogin(loginList) ?? '',
+        state: draftValues.state ?? state,
+        zipPostCode: draftValues.zipPostCode ?? zip,
```

Comment:
This will cause regression. draftValues can be empty string.

Lets hold this PR for #33641